mirrord-operator/values.yaml

```diff
@@ -34,8 +34,12 @@ operator:
 
 agent:
   ## example
-  ## image: your-internal.registry.io/metalbear-co/agent
-  image: ""
+  ## image: 
+  ##   registry: "your-internal.registry.io/metalbear-co/agent"
+  ##   tag: "latest"
+  image:
```

Comment:
This wouldn't do anything. Need to edit config-map.yaml to use the new values (if set)

Reply:
Changed!